src/ray/common/ray_config_def.h

```diff
@@ -137,8 +137,8 @@ RAY_CONFIG(int, object_manager_repeated_push_delay_ms, 60000)
 /// chunks exceeds the number of available sending threads.
 RAY_CONFIG(uint64_t, object_manager_default_chunk_size, 1000000)
 
-/// Number of workers per process
-RAY_CONFIG(int, num_workers_per_process, 1)
+/// Number of workers per Java worker process
+RAY_CONFIG(int, num_workers_per_process_java, 1)
```

Comment:
Also add a `num_workers_per_process_python` and set it to 1?

Reply:
Done.